README

```diff
@@ -32,25 +16,30 @@ layer for Message Passing ([MPI](https://www.mpi-forum.org/)),
 [PGAS](http://www.pgas.org/)/[OpenSHMEM](http://www.openshmem.org/)
```

Comment:
optimized communication layer -> optimized communication framework.

I also don't think the above reflects up to date scope of UCX. We need to work a bit more on this definition. It does not really reflect ML, AI, and data analytics, etc.

Reply:
@shamisp the wording has not changed in this PR.
WDYT about taking it from RTD:
https://openucx.readthedocs.io/en/master/#openucx

Reply:
IMHO RTD is better

Reply:
@shamisp fixed, can u pls look?

Reply:
Look good. @tonycurtis can you please take a look at the PR.